test/webserv/integration/test_http_delete.py

```
@@ -0,0 +1,171 @@
+def create_file(file_path, content=""):
+    if file_path is None:
+        return
+
+    try:
+        os.makedirs(os.path.dirname(file_path), exist_ok=True)
+        with open(file_path, "w") as f:
+            f.write(content)
+        print(f"Created file: {file_path}")
+    except Exception as e:
+        print(f"Error creating file: {file_path}, {e}")
+        raise AssertionError
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)